google/cloud/security/inventory/inventory_pipeline_configs.yaml

```diff
@@ -0,0 +1,66 @@
+
+- resource: bigquery_datasets
+  module_name: load_bigquery_datasets_pipeline
+  depends_on: projects
```

Comment:
Can we avoid putting the onus of knowing the dependency on the user creating a config and store this statically somewhere? We'll know better than the user.